### PR TITLE
Better salt for user

### PR DESCRIPTION
**Pull request by Wardormeur**
also, doesn't crash on postgres when NULL byte is generated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,8 @@
     "async": "0.9.0",
     "cp-i18n-lib": "git+https://github.com/CoderDojo/cp-i18n-lib.git",
     "cp-logs-lib": "git://github.com/CoderDojo/cp-logs-lib#v1.0.1",
+    "cp-permissions-plugin": "git://github.com/CoderDojo/cp-permissions-plugin#0.0.1",
+    "crypto": "0.0.3",
```
Comment (DanielBrierton) [inline package.json]: crypto is a part of node, this dependency is a 5 year old implementation no longer maintained. Should be removed.

**Comment by DanielBrierton** [comment]
Other than my comment and the semistandard error, I'm happy to no longer have NULL bytes messing things up, so 👍

**Comment by butlerx** [comment]
Aside from semistandard and and not using built in crypto looks good to me

**Comment by butlerx** [comment]
Tested works

**Pull request merged**